packages/autorest.typescript/test/rlcIntegration/generated/multipleUrlParameters/package.json

```diff
@@ -73,10 +73,10 @@
     "generate:client": "echo skipped",
     "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
     "minify": "dev-tool run vendored uglifyjs -c -m --comments --source-map \"content='./dist/index.js.map'\" -o ./dist/index.min.js ./dist/index.js",
```

Comment:
@jeremymeng should we have another pr to remove the minify step: https://github.com/Azure/azure-sdk-for-js/pull/33841/files?

Reply:
Yes we should do that too.